UnityProject/Assets/Scripts/UI/Objects/Security/SecurityRecordsConsole/GUI_SecurityRecords.cs

```diff
@@ -87,7 +87,7 @@ public void UpdateIdText(NetLabel labelToSet)
 
 		public void LogIn()
 		{
-			if (console.IdCard == null || !console.IdCard.HasAccess(Access.security) || IsAIInteracting() == false)
+			if ((console.IdCard == null && IsAIInteracting() == false) || !console.IdCard.HasAccess(Access.security))
```

Comment:
![truth table](https://i.imgur.com/JuVuFhN.png)

Perhaps you want this?
```suggestion
			if ((console.IdCard == null || console.IdCard.HasAccess(Access.security) == false) && IsAIInteracting() == false)
```